OVERVIEW.md

```diff
@@ -148,6 +148,25 @@ jscs path[ path[...]] --preset=jquery
 
 In order to add/remove preset rules you will need to create a `.jscsrc` config file.
 
+### `--extract`
+With this option you can set file mask from which JavaScript should be extracted.
```

Comment:
I don't how to describe it right :-/
It's not about file extensions, but glob patterns, I guess.